onnx/reference/ops/_op_list.py

```diff
@@ -355,4 +318,8 @@ def load_op(
     return cl
 
 
+def _build_registered_operators():  # type: ignore
```

Comment:
```suggestion
def _build_registered_operators():
```

was the ignore needed?

Reply:
removed